sources/BREAK-AND-TRACE

```diff
@@ -708,25 +699,23 @@ IL:|{DSK}<usr>local>lde>lispcore>sources>BREAK-AND-TRACE.;1|)
      IL:ELSE (EVAL IL:EXP IL:ENV)))
 
 (XCL:DEFOPTIMIZER IL:BREAK1 (&OPTIONAL IL:EXP IL:WHEN IL:FN IL:COMS TYPE XCL:CONDITION)
-                                (WHEN IL:COMS (IL:PRINTOUT T "BRKCOMS no longer supported:" IL:COMS T
-                                                     ))
-                                `(FLET
-                                  (($BRKEXP$ NIL ,IL:EXP))
-                                  (IL:IF ,IL:WHEN
-                                      IL:THEN
-                                      (LET
-                                       (($POS$ (IL:STKNTH 0 ',IL:FN)))
-                                       (UNWIND-PROTECT
-                                           (XCL:DEBUGGER
-                                            :FORM
-                                            `(FUNCALL ',#'$BRKEXP$)
-                                            :ENVIRONMENT NIL :STACK-POSITION $POS$ :CONDITION
-                                            ,(OR XCL:CONDITION
-                                                 `(IL:LOADTIMECONSTANT (XCL:MAKE-CONDITION
-                                                                        'BREAKPOINT :FUNCTION
-                                                                        ',IL:FN))))
-                                           (IL:RELSTK $POS$)))
-                                    IL:ELSE ($BRKEXP$))))
+                            (WHEN IL:COMS (IL:PRINTOUT T "BRKCOMS no longer supported:" IL:COMS T))
+                            `(FLET
+                              (($BRKEXP$ NIL ,IL:EXP))
+                              (IL:IF ,IL:WHEN
+                                  IL:THEN
+                                  (LET
+                                   (($POS$ (IL:STKNTH 0 ',IL:FN)))
+                                   (UNWIND-PROTECT
+                                       (XCL:DEBUGGER
+                                        :FORM
+                                        `(FUNCALL ',#'$BRKEXP$)
+                                        :ENVIRONMENT NIL :STACK-POSITION $POS$ :CONDITION
+                                        ,(OR XCL:CONDITION `(IL:LOADTIMECONSTANT
+                                                             (XCL:MAKE-CONDITION 'BREAKPOINT 
+                                                                    :FUNCTION ',IL:FN))))
+                                       (IL:RELSTK $POS$)))
+                                IL:ELSE ($BRKEXP$))))
 
```

Comment:
I thought the main thing to fix in BREAK-AND-TRACE was that when you said (BREAK (MUMBLE :IN FOO)) that it was FOO that might be unsafe to break or advise and not MUMBLE.

Reply:
> I thought the main thing to fix in BREAK-AND-TRACE was that when you said (BREAK (MUMBLE :IN FOO)) that it was FOO that might be unsafe to break or advise and not MUMBLE.

There are more things to fix there, but break and debug are really very opaque now.  I wanted to do a simple thing:  put the name of the broken function in the title of the debugger window.  I found where the title appeared to be asserted, but I couldn't figure out how to do it.

In this case I think I was just trying to get the filemap to be accurate